### PR TITLE
docs: egress docs

### PR DESCRIPTION
**Pull request by shyogesh-sw**
Issue #, if available: GALI-891

Description of changes: Updated documentation for the egress feature.

Checklist:

- [ ] Have you successfully deployed to an AWS account with your changes?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully tested with your changes locally?
- [ ] If new dependencies have been added, have they been pinned to specific versions?
- [ ] Is this change also required on the AWS Solution version?
- [ ] Have you updated openapi.yaml if you made updates to API definition (including add, delete or update parameter and request data schema)?
- [ ] If you had to run manual tests, have you considered automating those tests by adding them to [end-to-end tests](../main/end-to-end-tests/README.md)?
- [ ] If you are updating the changelog and vending out a new release, have you updated versionNumber and versionDate in [.defaults.yml](../main/config/settings/.defaults.yml)

AS review ticket id:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/docs/deployment/post_deployment/account_structure.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 ---
 
 Service Workbench uses _three_ types of accounts. You will see these account names throughout the documentation. 
+
+- **Organizational** : Holds the AWS Organization which creates hosting accounts.  Note that you may already have a method of obtaining AWS accounts supported by your organization.  If this is the case, you will not create an organizational account or use the Create Account functionality within Service Workbench when onboarding a hosting account.
 - **Main**: The account from which Service Workbench is deployed. Will be billed for all AWS usage charges in this deployment.
-- **Master**: Holds the AWS Organization which creates Member accounts.
-- **Hosting**: User accounts created within Service Workbench for individuals.
+- **Master**: Holds the AWS Organization which creates member accounts.
+- **Hosting**: Accounts that are established associated to the Service Workbench main account through the onboarding process to host the compute resources (Amazon SageMaker notebook instances, Amazon EC2 Windows and Linux instances, Amazon EMR clusters) associated to Service Workbench workspaces.
```
Comment (nguyen102) [inline docs/docs/deployment/post_deployment/account_structure.md]: `Accounts that are associated to the Service Workbench main account through the onboarding process to host the compute resources (Amazon SageMaker notebook instances, Amazon EC2 Windows and Linux instances, Amazon EMR clusters).`

Also should we call `Hosting` as `Member` now? In line 11 we called this `member`

File: docs/docs/deployment/post_deployment/aws_accounts.md [hunk 2]
```diff
@@ -16,185 +16,197 @@ Every user is linked to an **Account** through a **Project** and an **Index**, s
...
+2. This opens a new tab in your browser. When prompted, log in as administrator. This opens the Windows Desktop that you can interact with to create your AppStream image.
+3. On the Windows Desktop, choose **Start** and enter `Windows Powershell`.
+4. Right-click the application and choose **Run as administrator**.
+5. Enter the following commands:
+     ```
+     \cd ~\Documents
 
-_**Figure 16: Acknowledgement window**_
+     # Pull the Image Builder script from Github
+     Invoke-WebRequest -Uri https://raw.githubusercontent.com/awslabs/service-workbench-on-aws/feat-secure-workspace-egress/scripts/app-stream/buildImage.ps1 -OutFile buildImage.ps1
```
Comment (nguyen102) [inline docs/docs/deployment/post_deployment/aws_accounts.md]: You can change this `feat-secure-workspace-egress` to `mainline`

Comment (shyogesh-sw) [reply docs/docs/deployment/post_deployment/aws_accounts.md]: Fixed. Thanks

File: docs/docs/deployment/post_deployment/account_structure.md (hunk 1 above)
Comment (shyogesh-sw) [reply docs/docs/deployment/post_deployment/account_structure.md]: Changed to "member". Thanks

**Review by nguyen102 — DISMISSED** [review]

**Review by nguyen102 — APPROVED** [review]

**Pull request merged**